lib/vet360/models/base.rb

```diff
@@ -11,6 +11,16 @@ class Base
       include Virtus.model(nullify_blank: true)
 
       SOURCE_SYSTEM = 'VETSGOV'
+
+      validate :past_date?
+
+      private
+
+      def past_date?
+        if effective_end_date.present?
+          errors.add(:effective_end_date, 'must be in the past') if effective_end_date > Time.zone.now
```

Comment:
No need to change this, but an alternative to `Time.zone.now` is `Time.current`.  They do the same thing :)

Reply:
✅